server/src/test/java/org/apache/lucene/geo/XTessellatorTests.java

```diff
@@ -0,0 +1,1103 @@
+            + "-94.090393 47.7919809, -94.0901788 47.7883992, -94.0956694 47.7883497, -94.0953154 47.7736816, -94.1009001 47.773599,"
+            + "-94.1007919 47.7773656, -94.1011601 47.7882999, -94.1224262 47.7882635, -94.1278201 47.7882045, -94.128157 47.791947,"
+            + "-94.1334767 47.7919893, -94.1340009 47.7990339, -94.1391714 47.7989574, -94.1392586 47.8025871, -94.1445192 47.802501,"
+            + "-94.1441646 47.7952601, -94.1495363 47.7952326, -94.1498918 47.8024807, -94.1553562 47.8024703, -94.1555675 47.806059,"
+            + "-94.150131 47.8061278, -94.1506011 47.8129253, -94.1501789 47.8129684, -94.149611 47.8132643, -94.1491352 47.813398,"
+            + "-94.1489002 47.8135939, -94.1488101 47.8138527, -94.1488665 47.8142391, -94.1483486 47.8146313, -94.1482663 47.8152448,"
+            + "-94.1483213 47.8155667, -94.1490605 47.817251, -94.1497062 47.8182566, -94.1498362 47.8193197, -94.1499652 47.8195646,"
+            + "-94.150403 47.8213749, -94.1507045 47.821924, -94.1508205 47.8222638, -94.151727 47.8234564, -94.15213 47.8236878,"
+            + "-94.1524748 47.8240132, -94.1531753 47.8251591, -94.1537606 47.8265457, -94.1541208 47.8270211, -94.1545129 47.8273013,"
+            + "-94.1546514 47.8275679, -94.1551127 47.8277094, -94.1552844 47.8278376, -94.1554978 47.828217, -94.1561502 47.8288194,"
+            + "-94.1562033 47.8290381, -94.1567283 47.8298772, -94.1570108 47.8302438, -94.1572449 47.8303308, -94.1579044 47.8312801,"
+            + "-94.1588831 47.8316117, -94.1589829 47.8317057, -94.1589776 47.8319543, -94.1592481 47.8324046, -94.1601486 47.8333603,"
+            + "-94.1612329 47.8342848, -94.1619696 47.8351853, -94.155575 47.8352963, -94.1556138 47.8388745, -94.144871 47.8390595,"
+            + "-94.144906 47.8426792, -94.1396281 47.8427276), (-94.1340009 47.7990339, -94.1288303 47.7991103, -94.1291281 47.8027466,"
+            + "-94.1342625 47.802665, -94.1340009 47.7990339), (-94.1013659 47.7924644, -94.1014295 47.792468, -94.102571 47.7924213,"
+            + "-94.1027201 47.7923202, -94.1027617 47.7920166, -94.1026243 47.7918031, -94.1024919 47.7911957, -94.102004 47.7901082,"
+            + "-94.1018445 47.7888476, -94.1017758 47.7887408, -94.101548 47.7886445, -94.1013617 47.7886284, -94.1006145 47.7891011,"
+            + "-94.1005105 47.7892095, -94.1004868 47.789344, -94.1007015 47.790076, -94.1011754 47.7905011, -94.1012329 47.7907064,"
+            + "-94.1011431 47.7908595, -94.1009054 47.7909244, -94.1007312 47.7908545, -94.1005935 47.7906231, -94.100326 47.7905361,"
+            + "-94.0999407 47.7905308, -94.0998106 47.7906151, -94.1000388 47.7906356, -94.0993399 47.790923, -94.0992874 47.7910474,"
+            + "-94.0994397 47.7913413, -94.099697 47.7915716, -94.0997034 47.7918759, -94.0998534 47.7920624, -94.1005516 47.7924138,"
+            + "-94.1013659 47.7924644))";
+        checkPolygon(wkt);
+    }
+
+    private String reformat(String line, String name) {
```

Comment:
This was used to reformat all WKT and GeoJSON strings to fit the ES checkstyle requirements while also doing line breaks between lat/lon pairs so humans like what they see.

We should delete it before committing. I left it here in case we add more tests, and want reformatting of those.